variables.tf

```diff
@@ -1,44 +1,42 @@
-variable "service" {
-  description = "Name of service"
-}
-
-variable "name" {
-  description = "Name of secret"
-  default     = ""
+variable "automatically_after_days" {
+  description = "Number of days between automatic scheduled rotations of the secret"
+  default     = 1
 }
 
 variable "description" {
   description = "Description of the secret"
-  default     = ""
 }
 
 variable "kms_key_id" {
   description = "AWS KMS customer master key (CMK) to be used to encrypt the secret"
-  default     = ""
+  default     = null
+}
+
+variable "lambda_function_name" {
+  description = "Name of lambda function to be used by Secret Manager for key rotation"
+  default     = "aes128-key-rot"
+}
+
+variable "name" {
```

Comment:
No default here. Is that because something has to be specified, so you don't even bother setting a default of null?